Exercise_E.js

```diff
@@ -34,7 +40,14 @@ var KEYS = {
 };
 
 function maya(number) {
-  // CODE HERE
-}
+  var results = []
+  var dan = number.toString().split("");
+  for (var i=0; i < dan.length; i++){
+      results += KEYS[dan[i]]    
+      }
+      return results
+    }
+
+  

```

Comment:
Nice job this logic is vey good. 
Consider your naming conventions variable names are meant to be obvious.